src/index.html

```diff
@@ -9,10 +13,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The href attribute in the link tag should point to a CSS file, not an SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML document. Please ensure the href points to a compiled CSS file.